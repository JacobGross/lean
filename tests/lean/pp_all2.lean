import data.nat

set_option pp.all       true
set_option pp.numerals  true
set_option pp.universes false

check (10 : nat) + (3 : nat)
