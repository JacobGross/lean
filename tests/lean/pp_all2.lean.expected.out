@add nat nat_has_add 10 3 : nat
